js/styles/atlas.css

```diff
@@ -299,6 +299,7 @@ pre span.key {
   border-bottom: solid 1px #ccc;
   white-space: nowrap;
   text-align: right;
+  overflow: hidden;
```

Comment:
I would work on the cause of the issue instead of fixing the consequences. What do you think?